src/librustc_error_codes/error_codes/E0399.md

```diff
@@ -1,35 +0,0 @@
-You implemented a trait, overriding one or more of its associated types but did
```

Comment:
Shouldn't this file be left alone with a note mentioning that it is no longer emitted?

Reply:
Sure, I can do that. Not sure what the proper process is here, but the code examples probably need to be `ignore`d then, if that even works on the error index files. https://github.com/rust-lang/rust/commit/e01ad6a01abce35f59543bf38a280a05eb7f6929 also removed the file directly.

Reply:
I believe the common action is to have a title "Note: this error code is no longer emitted by the compiler." at the top and ignore the code samples: https://doc.rust-lang.org/error-index.html